### PR TITLE
add chime-notifier

### PR DESCRIPTION
**Pull request by flabat**
This tool can be used to post alerts to a Chime romm when AWS Health events are generated by using AWS Lambda and Amazon CloudWatch Events. To learn more about Chime Webhooks, refer to [Automate Chat Messages with Webhooks](http://docs.aws.amazon.com/chime/latest/ug/webhooks.html)

### CODE REVIEW

File: chime-notifier/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+1. Click the cog wheel from your Chime room 
+
+2. Select "Manage WebHooks".
+
+3. Create Webhook
+
+4. Copy the webhook URL and use it in the AWS Setup section that follows.
+
+
```
Comment (mwpatrick) [inline chime-notifier/README.md]: A reminder to treat the webhook like a password (i.e. don't save it somewhere public) would be good

File: chime-notifier/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+More information about Lambda is available here: http://docs.aws.amazon.com/lambda/latest/dg/getting-started.html
+
+3. Create a CloudWatch Events rule to trigger the Lambda function created in step 2 for AWS Health events.
+Documentation on how to create an AWS Health CloudWatch Events rule is available here: http://docs.aws.amazon.com/health/latest/ug/cloudwatch-events-health.html
+
+More information about AWS Health is available here: http://docs.aws.amazon.com/health/latest/ug/what-is-aws-health.html
+
+Note that this is a just an example of how to set up automation with AWS Health, Amazon CloudWatch Events, and AWS Lambda. We recommend testing the example and tailoring it to your environment before using it in your production environment.
+
```
Comment (mwpatrick) [inline chime-notifier/README.md]: Do you have a sample test you could suggest to the user about how they could force a health event?
At a minimum there could be a link to the documentation at http://docs.aws.amazon.com/AmazonCloudWatch/latest/events/EventTypes.html#health-event-types which would show a structure for doing a test of their Lambda configuration at least.

**Comment by mwpatrick** [comment]
One other comment - Is it good to provide both JSON and YAML CloudFormation templates? The thought which comes to mind is that in the future that is now three places where the Python function code has to be updated and two test paths. My suggestion is just to do the YAML (since it allows inline comments in the CFN template).

File: chime-notifier/README.md (hunk 2 above)
Comment (flabat) [reply chime-notifier/README.md]: Updated README.md with clarification on how to manually test and a link to the Health test event page.

File: chime-notifier/README.md (hunk 1 above)
Comment (flabat) [reply chime-notifier/README.md]: updated the readme

File: chime-notifier/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+### AWS Manual Setup
+
+1. Create an IAM role for the Lambda function to use. Attach the [IAM policy](IAMPolicy) to the role in the IAM console.
+
+Documentation on how to create an IAM policy is available here: http://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html
+Documentation on how to create an IAM role for Lambda is available here: http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-service.html#roles-creatingrole-service-console
+
+2. Create a Lambda Python function by using the [sample](LambdaFunction.py) provided and choose the IAM role created in step 1. Add an enviroment variable with key CHIMEWEBHOOK and the webhook URL from the Chime setup above as value.
```
Comment (dstroppa) [inline chime-notifier/README.md]: When you create a Python Lambda function the handler defaults to 'lambda_function.lambda_handler'. In the sample function code provided the handler is called 'handler'. Either update the code to use 'lambda_handler' or add a step here to change the handler name.

Comment (flabat) [reply chime-notifier/README.md]: updated to lambda_handler

Comment (flabat) [reply chime-notifier/README.md]: updated to lambda_handler

**Pull request merged**